tests/testthat/test-call_derivation.R

```diff
@@ -2,8 +2,10 @@ library(admiral.test)
 data(admiral_ae)
 data(admiral_vs)
 
-test_that("call_derivation works", {
-  input <- admiral_vs[sample(seq_len(nrow(admiral_vs)), 1000), ]
+## Test 1: Test that call_derivation generates expected summary output  ----
+# ---- call_derivation, test 1: call_derivation Test 1:  Test that call_derivation generates expected summary output ----
+test_that("call_derivation, test 1: call_derivation Test 1:  Test that call_derivation generates expected summary output", {
+  input <- admiral.test::admiral_vs[sample(seq_len(nrow(admiral.test::admiral_vs)), 1000), ]
```

Comment:
Hey @jeffreyad.  Sorry this one is taking a while.  Can you remove the `library(admiral.test)` and `data(...)` at the top of the test file and replace all the calls for` admiral_vs`, etc with `admiral.test::admiral_vs` and the similar calls.  Looks like I committed the first one on accident, but we should update all the other tests below.  This is considered best practices by the R package overlords now.